.storybook/preview.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Inter } from "next/font/google";
+import type { Preview } from "@storybook/react";
+import "../src/app/globals.css";
+import { L10nProvider } from "../src/contextProviders/localization";
+import { getL10nBundles } from "../src/app/functions/server/l10n";
+import { metropolis } from "../src/app/fonts/Metropolis/metropolis";
+
+const inter = Inter({ subsets: ["latin"], variable: "--font-inter" });
```

Comment:
Question/Suggestion (for another PR?): Should we bring in the fonts the same way, instead of importing one from Google and one locally? My sense is to have the fonts all serve the same way/location for reliability.

Reply:
Yeah good question. I don't have a strong opinion; I like the consistency, but I also like automatically getting the latest/most complete version, and not vendoring dependencies where possible. But I'd approve a PR that changes it to a local font :)